hooks/__tests__/useCopyToClipboard.test.tsx

```diff
@@ -0,0 +1,83 @@
+import { render, fireEvent, screen, waitFor } from '@testing-library/react';
+import { FormattedMessage } from 'react-intl';
+import { useCopyToClipboard } from '../useCopyToClipboard';
+import { LocaleProvider } from '../../providers/localeProvider';
+import type { AppProps } from '../../types';
+
+const i18nData = { currentLocale: { code: 'en' } } as AppProps['i18nData'];
+
+describe('useCopyToClipboard', () => {
+  const HookRenderer = ({ text }: { text: string }) => {
+    const [copied, copyText] = useCopyToClipboard();
+
+    return (
+      <LocaleProvider i18nData={i18nData}>
```

Comment:
Can you remove the locale provider from here?

Reply:
It's a must
```
[React Intl] Could not find required `intl` object. <IntlProvider> needs to exist in the component ancestry.
```

Reply:
Yes because you're passing FormattedMessage there. You can simply remove that altogether lol